public/stylesheets/style.css

```diff
@@ -110,3 +110,7 @@ body{
 .form__help-block--hidden {
   display: none;
 }
+
+.starred {
```

Comment:
@jrob8577 so the only method I can think of to match workflowy's style is to use the actual svg files that workflowy uses for selected and unselected stars. Is that what you're looking for? Or should I modify the star in our font library to look similar but not exactly the same?